packages/deployment/app/cloudformation.template.yml

```diff
@@ -935,7 +935,22 @@ Resources:
               echo 'ECS_ENGINE_AUTH_DATA={"https://index.docker.io/v1/":{"username":"{{ecs.docker.username}}","password": "${DockerPassword}","email":"${DockerEmail}"}}' >> /etc/ecs/ecs.config
               {{/ifEquals}}
 
-              --==BOUNDARY==--
+              yum install -y jq unzip
+              curl "https://s3.amazonaws.com/aws-cli/awscli-bundle.zip" -o "awscli-bundle.zip"
+              unzip awscli-bundle.zip
+              ./awscli-bundle/install -i /usr/local/aws -b /usr/local/bin/aws
+              rm -rf ./awscli-bundle awscli-bundle.zip
+
+              aws s3 cp s3://{{bucket}}/{{stackName}}/deployment-staging/task-reaper.sh /usr/local/bin/task-reaper.sh
+              chmod +x /usr/local/bin/task-reaper.sh
+
+              cat <<'EOF' >> /etc/cron.d/task-reaper
+              PATH=/bin:/usr/local/bin
```

Comment:
should this be something like `PATH= /bin:/usr/local/bin/$PATH`?

Reply:
No, the path specified here works.